proxy/envoy/discovery.go

```diff
@@ -515,20 +528,21 @@ func (ds *DiscoveryService) ListSecret(request *restful.Request, response *restf
 	_, secret := buildIngressRoutes(ds.Mesh, ds, ds)
 
 	if secret == "" {
+		glog.V(3).Infof("Secret is not set")
```

Comment:
I agree. May be increase log level to 6 or something? Generally, everything in the *DS path loops needs to have high log level.. So that it is easier to debug in stages..